src/player.scss

```diff
@@ -65,6 +65,19 @@ h1.md-headline {
 	min-width: 200px;
 }
 
+.dropdown-answer-element {
+	position: relative;
+
+	.answer-number {
+		position: absolute;
+		right: 0;
+		bottom: 0;
```

Comment:
IMO, stylistically, would it be better to center-align this text, instead of aligning it with the bottom of the current item of the list?
![screen shot 2019-02-04 at 2 30 12 pm](https://user-images.githubusercontent.com/1268547/52232029-69abad00-2889-11e9-8958-628d9b25d6f2.png)

Besides that though, LGTM. 👍

Reply:
It is done.